playbooks/roles/edxapp/templates/edx/app/edxapp/lms.sh.j2

```diff
@@ -17,7 +17,18 @@ if command -v ec2metadata >/dev/null 2>&1; then
   HOSTNAME=$(hostname)
   export NEW_RELIC_PROCESS_HOST_DISPLAY_NAME="$HOSTNAME-$INSTANCEID"
 fi
+
+
+{% if COMMON_DEPLOYMENT == "edge" %}
+# Another attempt at a temporary workaround for trace concatenation in Datadog.
+# (Maybe this will keep NR trace instrumentation in place, but not cost us anything,
+# because no data will be shipped to NR.)
+# See https://github.com/edx/edx-arch-experiments/issues/692
+export NEW_RELIC_LICENSE_KEY="xxx"
```

Comment:
Also, do we want an invalid key or an empty key?

Reply:
Invalid key is what I tested with in devstack. Definitely want to override it with something just in case there's another environment variable lurking somewhere.